### DISCUSSION TITLE
Release date for 4.3.2

### DISCUSSION THREAD

**Discussion by faisal-shah** [hardbyte/python-can#1741] [General]
What's the anticipated release date for 4.3.2?

**Comment by hardbyte** [comment]
I've started a branch and drafted release notes here https://github.com/hardbyte/python-can/pull/1759

**Comment by hardbyte** [comment]
Pre-release now on PyPI - please try it out https://pypi.org/project/python-can/4.3.2rc4/

```
pip install python-can==4.3.2rc4
```
